modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js

```diff
@@ -24,6 +24,26 @@ import './Modal.scss';
 import delegate from '../delegate/delegate.es';
 import navigate from '../util/navigate.es';
 
+const alert = ({message}) => {
+	openModal({
+		bodyHTML: message,
+		buttons: [
+			{
+				label: Liferay.Language.get('ok'),
+				onClick: ({processClose}) => {
+					processClose();
+				},
+				otherProps: {
```

Comment:
It will require I made on https://github.com/liferay-frontend/liferay-portal/pull/1988/files#r821204218

Reply:
Thanks for the explanation, @markocikos! Changing stuff here... :)